src/OpenCensus.Collector.AspNetCore/RequestsCollector.cs

```diff
@@ -53,11 +53,10 @@ public RequestsCollector(RequestsCollectorOptions options, ITracer tracer, ISamp
                     {
                         s = options.CustomSampler(x);
                     }
-                    catch (Exception)
+                    catch (Exception e)
                     {
                         s = null;
-
-                        // TODO: Add error logging
+                        AspNetCoreCollectorEventSource.Log.ExceptionInCustomSampler(e.Message, e.StackTrace);
```

Comment:
please check out how exception can be logged in English locale. Just copy method from other event source

Reply:
Please address this. Ideally make this locale transformation to be part of EventSourece class and only pass an exception object to the method. This will also improve performance as when event source is not enabled - there is no perf impact on serializing text

Reply:
Done